TYPO3.Flow/Classes/TYPO3/Flow/Command/DoctrineCommandController.php

```diff
@@ -321,6 +337,18 @@ public function migrationExecuteCommand($version, $direction = 'up', $output = N
 	}
 
 	/**
+	 * @param \Exception $exception
+	 * @param string $version
+	 */
+	protected function handleMigrationException(\Exception $exception, $version) {
+		$this->outputLine($exception->getMessage());
+		$this->outputLine();
+		$this->outputLine('Something wrong happen during the migration: ' . $version);
```

Comment:
"An exceptional error occurred during the migration %s. The exception details have been logged to the Flow system log."